src/AppInstallerCLICore/Workflows/MultiQueryFlow.cpp

```diff
@@ -80,7 +80,7 @@ namespace AppInstaller::CLI::Workflow
             case SearchPurpose::Uninstall:
                 searchContext <<
                     Workflow::HandleSearchResultFailures <<
-                    Workflow::EnsureOneMatchFromSearchResult(true);
+                    Workflow::EnsureOneMatchFromSearchResult(SearchPurpose::Uninstall);
                 break;
             default:
                 THROW_HR(E_UNEXPECTED);

```

Comment:
Would any of the new enum values be something that we can handle here?

Reply:
I’m not sure I understand

Reply:
I meant, should this function handle any of Default, Export, List, RequiresInstalledSource?

Reply:
Ah, yes, EnsureOneMatchFromSearchResult does handle any of the enum values. For any of the values like List/Export where an installed package is required, it will use the "No installed package found. . ." message, for upgrade it uses the "No available upgrades" message, and for anything from a source like install it will use the "No package found matching criteria" message

Reply:
Should SearchSubContextsForSingle also handle those? Looking at the other values, it doesn't seem to me like it needs to. For example, I don't think we will have to search on sub-contexts like this for export. But I wanted to see what you thought.

Reply:
I don’t think we will need to unless there's a request for exporting only specific packages, which would require a multiquery and subcontexts to aggregate all of the packages. Future work is supported if needed, but I assumed that continuing to throw the UNEXPECTED termination code would be enough for now